src/exec/graphics.rs

```diff
@@ -0,0 +1,204 @@
+// Copyright 2022 System76 <info@system76.com>
+// SPDX-License-Identifier: GPL-3.0-only
```

Comment:
Needs to be MPL-2.0 here, and other files should also have MPL-2.0 SPDX license headers.